app/views/content/non-uk-teachers/visas-for-non-uk-teachers.md

```diff
@@ -57,7 +61,19 @@ keywords:
 
 If you’re applying for teaching jobs in England as a non-UK citizen, you will need a [visa or immigration status allowing you to work in England](#check-your-status).
 
-Most non-UK citizens who don't already have the correct visa or immigration status will need to apply for a Skilled Worker visa to teach in England. 
+Most non-UK citizens who do not already have the correct visa or immigration status will need to apply for a Skilled Worker visa to teach in England. 
+
+<section class="inset-text purple">
```

Comment:
This looks fine on the front-end but I thought we normally defined these insets in the frontmatter and then called them in the body?

Paging @emmaachesongray for advice!